app.py

```diff
@@ -47,13 +96,12 @@ def create_post():
 
         # check if the form should parse the stuff or should get as dictionary
         form_info = request.form  # dictionary of form data
-        
+
         timestamp = datetime.now().strftime("%Y-%m-%d %H:%M")
         
         # Things to add:
         # Post needs to refer to session or something to get the actual sid instead of manual input
-        #sid = request.form.get('sid')
-        # helper.add_post(conn, title, description, timestamp, location, tag, professor, course, None)
+        # sid = request.form.get('sid')
```

Comment:
I'll take a look at this

Reply:
Thanks, can probably update that after login is implemented!